ecosystem/sep-0006.md

```diff
@@ -420,7 +424,7 @@ The wallet must use the response fields in the following way to complete the wit
 - `withdraw_anchor_account`: send the withdrawal payment to this Stellar account. 
 - `withdraw_memo`: (if specified) use this memo in the payment transaction to the anchor.
 - `withdraw_memo_type`: use this as the memo type.
-- `amount_in`: the amount expected in the stellar payment.
+- `amount_in`: the amount expected in the Stellar payment (with an acceptable error margin of 10%)
```

Comment:
The amount_in should never fluctuate right?  Only the amount that actually ends up sent? The error margin here is confusing.

Reply:
Yeah, I agree it's confusing. I wanted to make it clear for wallets that there is an acceptable error margin, but this is probably not the spot. I'll push a change to address this shortly.